README.md

```diff
@@ -61,24 +62,29 @@ This is especially relevant for supporting dedicated worker pools for shoot `etc
 
 The steps for installing kupid on a Kubernetes cluster for development and/or trial are given below.
 These are only development installation steps and not intended for any kind of production scenarios.
-For anything other than development or trial purposes, please use your favourite CI/CD toolkit.
+For anything other than development or trial purposes, please use your favorite CI/CD toolkit.
```

Comment:
Not really necessary to change this, since it's just a difference between American and British spelling.

Reply:
Yes that's true - the quick fix tool that was used to fix grammar autocorrected it and I didn't notice this particular change. Sorry about that.